assets/src/stories-editor/blocks/amp-story-page-attachment/attachment-opener.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+import { ENTER, SPACE } from '@wordpress/keycodes';
+
+const AttachmentOpener = ( { setAttributes, toggleAttachment, openText } ) => {
+	return (
+		<div className="open-attachment-wrapper">
+			<span
+				role="button"
```

Comment:
Curious: why did a regular `button` not work?

Reply:
It might have, this was a replication of the front-end markup how the Page Attachment displays it. Do you have a preference for trying it out with `button` instead?

Reply:
Not urgent to change it I suppose.